charts/operator/deployment.go

```diff
@@ -230,6 +249,60 @@ func operatorPodVolumes(dot *helmette.Dot) []corev1.Volume {
 			},
 		},
 	}
+
+	if values.ServiceAccount.Create && !ptr.Deref(values.ServiceAccount.AutomountServiceAccountToken, false) {
```

Comment:
Do you know what would happen if `AutomountServiceAccountToken` was true and we added this mount?

Not pressing, just curious.

Reply:
I didn't test that, but looking at the source code if volumes and volumesMounts follow naming scheme, then admission logic will not auto mount. 

https://github.com/kubernetes/kubernetes/blob/66e34012255abf1bbd0956a712817dad77c69c41/plugin/pkg/admission/serviceaccount/admission.go#L416-L482